deployment/helm/human-connection/templates/deployments/deployment-neo4j.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name:  {{ .Release.Name }}-neo4j
+  labels:
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/name: human-connection
+    app.kubernetes.io/version: {{ .Chart.AppVersion }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: "100%"
+  selector:
+    matchLabels:
+      human-connection.org/selector: deployment-neo4j
+  template:
+    metadata:
+      name: neo4j
+      annotations:
+        backup.velero.io/backup-volumes: neo4j-data
+      labels:
+        human-connection.org/commit: {{ .Values.commit }}
+        human-connection.org/selector: deployment-neo4j
+    spec:
+      containers:
+      - name: neo4j
+        image: "{{ .Values.neo4jImage }}:{{ .Chart.AppVersion }}"
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-04T13:07:31Z" title="Tuesday, February 4th 2020, 2:07:31 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -33,15 +38,6 @@ spec:
             memory: {{ .Values.neo4jResourceRequestsMemory | default "1G" | quote }}
           limits:
             memory: {{ .Values.neo4jResourceLimitsMemory | default "1G" | quote }}
-        env:
-        - name: NEO4J_dbms_security_procedures_unrestricted
-          value: "algo.*,apoc.*"
```


Obsolete? We don't need that anymore?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-04T13:26:03Z" title="Tuesday, February 4th 2020, 2:26:03 pm +01:00">Feb 4, 2020</time>_

---


we do need it actually! I ran into an issue with running the `yarn prod:migrate init` yesterday on the `develop` server because this was removed... I'll add it back.

I did have a question about whether we should have `production` config stored locally and `.gitignored`, or how we should handle that? I would like to have a copy of what is being deployed in production in case there is a need to recreate the cluster for some reason.

I have looked into `helm-secrets` and got it installed on my computer, but when I tried to use it, it threw errors. Before debugging, I though it would be best to come to some consensus on it.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-04T14:48:35Z" title="Tuesday, February 4th 2020, 3:48:35 pm +01:00">Feb 4, 2020</time>_

---


Actually, I hadn't realized that you are `.gitignore`ing the `configmap.yaml` and `secrets.yaml` ... I have extracted all the `env` variables there.
My suggestion would be for myself and you to have different files like `develop.configmap.yaml`/`production.configmap.yaml` or different `values.yaml` and `.gitignore` them.